src/amqp_manager.ts

```diff
@@ -124,7 +125,7 @@ export class AmqpManager extends EventEmitter {
 
          setTimeout(() => {
             reject(new Error('Channel Timeout'))
-         }, 2000)
+         }, this.config.channel_timeout)
```

Comment:
Perhaps default this to 2000 since it’s optional. That will keep it functionally backwards compatible.